Gemfile.lock

```diff
@@ -207,26 +207,28 @@ GEM
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
     mercenary (0.3.6)
-    mini_portile2 (2.8.1)
+    mini_portile2 (2.8.4)
     minima (2.5.1)
       jekyll (>= 3.5, < 5.0)
       jekyll-feed (~> 0.9)
       jekyll-seo-tag (~> 2.1)
     minitest (5.19.0)
-    nokogiri (1.14.3)
-      mini_portile2 (~> 2.8.0)
+    nokogiri (1.15.4)
```

Comment:
Unfortunately, GItHub Pages 228 (latest version) mandates nokogiri to be < 2.0, while the vulnerabilty would be fixes only in its 2.9.6 version according to the alert. Still according to [GitHub Pages dependencies](https://pages.github.com/versions/), nokogiri should be at its 1.15.2 version. Maybe that will still fix the security vulnerability.